ayushma/models/project.py

```diff
@@ -16,6 +16,9 @@ class Project(BaseModel):
     stt_engine = models.IntegerField(
         choices=STTEngine.choices, default=STTEngine.WHISPER
     )
+    tts_engine = models.IntegerField(
```

Comment:
SmallInteger would be more suitable here

Reply:
Agreed